### PR TITLE
Runrst

### LINKED ISSUE #1263: run.rst: Convert to py interface

### PR DESCRIPTION
**Pull request by jonaslandsgesell**
Closes #1263.

### CODE REVIEW

File: doc/sphinx/run.rst [hunk 1]
```diff
@@ -46,13 +45,13 @@ would like to recompute the forces, despite the fact that they are
 already correctly calculated. To this aim, the option can be used to
 enforce force recalculation.
 
-``minimize_energy``: Run steepest descent minimization
```
Comment (KaiSzuttor) [inline doc/sphinx/run.rst:49]: Why not to have the command in the heading?

Comment (jonaslandsgesell) [reply doc/sphinx/run.rst:49]: To my understanding the command names are part of the inline documentation.

**Comment by KaiSzuttor** [comment]
Error compiling Cython file:
------------------------------------------------------------
...

        """
        return self.box_l[0] * self.box_l[1] * self.box_l[2]

    def distance(self, p1, p2):
        """Return the scalar distance between the particles, respecting periodic boundaries.

**Comment by KaiSzuttor** [comment]
Closes #1267.

**Review by hmenke — DISMISSED** [review]
It doesn't look like you checked whether any tests or samples still rely on `remove()`.

**Comment by jonaslandsgesell** [comment]
Tests worked

Am 21.09.2017 11:31 Nachm. schrieb "Henri Menke" <notifications@github.com>:

**Comment by hmenke** [comment]
No, they don't.  (Sorry, I meant `delete()`, not `remove()`)

https://github.com/espressomd/espresso/blob/8f72173f47a5752ccf36b81b76abe493bbbb63fc/testsuite/mass-and-rinertia_per_particle.py#L30

**Comment by jonaslandsgesell** [comment]
Strange. I see your point. I ran make check and it worked though.

Am 22.09.2017 1:37 Vorm. schrieb "Henri Menke" <notifications@github.com>:

**Review by fweik — APPROVED** [review]

**Pull request merged**